### PR TITLE
Inv updates

### PR DESCRIPTION
**Pull request by santoshpuranik**

### CODE REVIEW

**Comment by geissonator** [comment]
add to approvelist

**Comment by geissonator** [comment]
jenkins run tests please

**Comment by santoshpuranik** [comment]
jenkins run tests please

**Review by RameshIyyar — COMMENTED** [review]
LGTM, Tested with Guard patches.

File: meta-ibm/recipes-phosphor/vpd/ibm-vpd-parser/50001000.json [hunk 1]
```diff
@@ -500,264 +490,768 @@
...
-      }
-    ],
-    "/sys/bus/spi/drivers/at25/spi22.0/eeprom": [
+      },
       {
-          "inventoryPath": "/system/chassis/motherboard/cpu1",
-          "powerOffOnly" : true,
-          "offset": 196608,
-          "size": 65504,
+          "inventoryPath": "/system/chassis/motherboard/dcm0/cpu0/unit0",
```
[the remark this answers is not in the document]
Comment (santoshpuranik) [reply meta-ibm/recipes-phosphor/vpd/ibm-vpd-parser/50001000.json:495]: They can't - this is sort of a hack to just cover our guard requirements. Ideally we should have had unique Inventory.Item interfaces. There does not seem to be use-case (yet) for anything other than just displaying them.

**Comment by rfrandse** [comment]
in GHE

**Pull request closed without merging**